tools/common/Application.cs

```diff
@@ -21,6 +21,12 @@
 
 namespace Xamarin.Bundler {
 
+	public enum BuildTarget {
+		None,
```

Comment:
If it's used for the Mac/host then let's give it that name, because `None` does not reflect the target.